content/_index.html

```diff
@@ -56,9 +56,10 @@ <h1 id="display">UNSW CPMSoc</h1>
     <div class="columns mt-6">
         <div class="column is-8 is-12-mobile">
             <h2>Stay Updated</h2>
-            <p>Follow us on <strong>Facebook</strong> to receive regular updates on all our exciting events and workshops. You can also get updates on our <strong>Discord</strong>, and join in on the discussion too!</p>
-            <a href="https://www.facebook.com/CPMSoc/" class="button is-link mr-3 mt-3">Facebook page</a> 
-            <a href="https://discord.gg/9uxgxHY3pP" class="button is-link mt-3">Discord server</a>
+            <p>Follow us on <strong>Facebook</strong> and our <strong>Google Calendar</strong> to receive regular updates on all our exciting events and workshops. You can also get updates on our <strong>Discord</strong>, and join in on the discussion too!</p>
```

Comment:
"Follow us on Facebook and our Google Calendar" -> "Follow us on Facebook and subscribe to our Google Calendar"

Also "our Discord" -> "our Discord server" while you're at it